### ISSUE TITLE
Support for Regex missing

### ISSUE THREAD

**Issue by mmanela** [krn-robin/tree-sitter-magik#48]
The grammar does not handle regexes. For example the following code

```magik
_if /Hello\,\s(\w)+!/.matches?("Hello, Magik!") _then
    write("Got a match!")
_endif
```

Results in the following AST with errors
![image](https://github.com/krn-robin/tree-sitter-magik/assets/304410/de174397-6f6e-46a0-b0b1-43b0df2ee8f9)

**Comment by krn-robin** [comment]
Implemented with #51 (d66edfd84d2751836d401097ff08711b1337548d)

**Comment by krn-robin** [comment]
Implemented with #51 ( d66edfd84d2751836d401097ff08711b1337548d )

**Comment by krn-robin** [comment]
Implemented with #51 ( d66edfd84d2751836d401097ff08711b1337548d )
